include/drivers/misc/ft8xx/ft8xx_memory.h

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (c) 2020 Hubert Miś
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief FT8XX memory map
+ */
+
+#ifndef ZEPHYR_DRIVERS_MISC_FT8XX_FT8XX_MEMORY_H_
+#define ZEPHYR_DRIVERS_MISC_FT8XX_FT8XX_MEMORY_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief FT8xx memory addresses
+ * @defgroup ft8xx_memory FT8xx memory map
+ * @ingroup ft8xx_interface
+ * @{
+ */
+
+#if defined(CONFIG_FT800)
+/** Main parts of FT800 memory map */
+enum ft8xx_memory_map_t {
+	RAM_G         = 0x000000,
```

Comment:
I think all these enums should be prefixed, ie `FT8XX_`

Reply:
I prefixed with `FT800_` and `FT810_`, because memory map differs between these devices.